auth-saml-ldap/package.json

```
@@ -0,0 +1,41 @@
+{
+  "private": true,
+  "sideEffects": false,
+  "scripts": {
+    "build": "remix build",
+    "dev": "npm run saml:cert; run-p dev:*",
+    "dev:server": "remix dev",
+    "dev:ldap": "docker run  -p 389:1389 -v \"$(pwd)/test/ldap/config\":/ldifs bitnami/openldap:latest",
+    "dev:saml": "node ./test/saml-idp/bin/run.js --acsUrl http://localhost:3000/auth/asc --audience http://localhost:3000/login",
+    "saml:cert": "openssl req -x509 -new -newkey rsa:2048 -nodes -subj '/C=US/ST=California/L=San Francisco/O=JankyCo/CN=Test Identity Provider' -keyout test/saml-idp/idp-private-key.pem -out test/saml-idp/idp-public-cert.pem -days 7300",
+    "saml:setup": "rm -rf test/saml-idp; mkdir test/saml-idp; git clone https://github.com/mcguinness/saml-idp.git test/saml-idp; cd test/saml-idp/; npm install",
+    "start": "remix-serve build",
+    "setup": "prisma generate && prisma migrate dev --name \"initial\""
```

Let's add `typecheck` script to be in line with all other examples (see #98)

```suggestion
    "setup": "prisma generate && prisma migrate dev --name \"initial\"",
    "typecheck": "tsc"
```